### PR TITLE
i18n: FR Translation update (9) (Ultimate)

### PR DESCRIPTION
**Pull request by mooondark**

### CODE REVIEW

File: ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts [hunk 1]
```diff
@@ -224,20 +224,23 @@ const triggerSet: TriggerSet<Data> = {
...
           de: 'Schwur auf ${player}',
+          fr: 'Vœu sur ${player}',
           ja: '${player}に滅殺',
           cn: '毒点 ${player}',
           ko: '${player} 멸살의 맹세',
         },
         vowSoon: {
           en: 'Vow soon (Spread)',
           de: 'Schwur bald (verteilen)',
+          fr: 'Vœu bientôt (Écartez-vous)}',
```
Comment (xiashtra) [inline ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:243]:
```suggestion
          fr: 'Vœu bientôt (Écartez-vous)',
```

File: ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts [hunk 2]
```diff
@@ -2284,13 +2356,15 @@ const triggerSet: TriggerSet<Data> = {
           tanksApart: {
             en: 'Apart (Hrae buster)',
             de: 'Auseinander (Hrae-buster)',
+            fr: 'Séparation Hrae buster)',
```
Comment (xiashtra) [inline ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:2359]:
```suggestion
            fr: 'Séparation (Hrae buster)',
```

File: ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts [hunk 3]
```diff
@@ -1694,6 +1759,7 @@ const triggerSet: TriggerSet<Data> = {
           noDebuff: {
             en: '(no debuff)',
             de: '(kein Debuff)',
+            fr: '(sans debuff',
```
Comment (xiashtra) [inline ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1762]:
```suggestion
            fr: '(sans debuff)',
```

**Review by xiashtra — CHANGES_REQUESTED** [review]
Some missing/excess parenthesis/braces in a few places.

**Comment by mooondark** [comment]
You're right, thanks for proofreading me ! 👍

**Review by xiashtra — APPROVED** [review]

**Pull request merged**